recipes/celerite2/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "celerite2" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9f469b2ad42b6d47f7cbdf9242ee637c7ed3954a97175475b11e502846316145
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
```

Comment:
This package looks like it's using/requires `setuptools_scm`, so I'm a little confused why the CI isn't complaining since it's not listed here.

Reply:
Good point and I do not know why!